yarn.lock

```diff
@@ -19452,7 +19301,7 @@ minimist@^1.1.1, minimist@^1.2.0, minimist@^1.2.5, minimist@^1.2.6:
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.6.tgz#8637a5b759ea0d6e98702cfb3a9283323c93af44"
   integrity sha512-Jsjnk4bw3YJqYzbdyBiNsPWHPfO++UGG749Cxs6peCu5Xg4nrena6OVxOYxrQTqww0Jmwt+Ref8rggumkTLz9Q==
 
-minimist@^1.2.8:
+minimist@^1.1.3, minimist@^1.2.8:
```

Comment:
1.1.3 minimist will cause CVE. need to be at least 1.2.6 version

Reply:
Resolved by using package resolution on minimist